pkg/activator/net/revision_backends_test.go

```diff
@@ -407,11 +406,12 @@ func TestRevisionWatcher(t *testing.T) {
 
 			waitInformers, err := controller.RunInformers(ctx.Done(), informer.Informer())
 			if err != nil {
-				t.Fatalf("Failed to start informers: %v", err)
```

Comment:
Any reason for this change?
I get that it is equivalent but it doesn't seem idiomatic, see:
https://github.com/golang/go/search?q=t.Fatalf&unscoped_q=t.Fatalf
https://github.com/golang/go/search?q=t.Fatal&unscoped_q=t.Fatal

Reply:
Saves cpu cycles, we probably need Google cloud resources for other stuff 🙄

Reply:
In the tests, in a code path pretty much never exercised 😆 ?

Reply:
You never know :) preloading both possible paths.
In other words, we're losing nothing :)